numba_dppy/dppy_debuginfo.py

```diff
@@ -0,0 +1,16 @@
+from numba.core.debuginfo import DIBuilder
+
+
+class DPPYDIBuilder(DIBuilder):
+    def mark_subprogram(self, function, name, loc):
+        if isinstance(name, tuple):
```

Comment:
See below. Tuple could be replaced by using `DPPYDIBuilder` member for `mangled_qualname`.